Assertion output regression

In master (https://github.com/sindresorhus/ava/commit/945dbea40a6a500a24d54ebcc7e10dcd75647fc7), there's no longer any `power-assert` output.

```js
import test from 'ava';

test(t => {
	t.ok(false);
});
```

```
  1 failed

  1. [anonymous]
  AssertionError: false == true
    at Test.fn (/Users/sindresorhus/dev/md5-hex/test.js:4:4)
```

Found this whilst writing a failing spec for a module I'm working on; gives this stack trace:

```
  1 passed  1 failed

  1. should return the smallest colour
  AssertionError: should convert percentage based rgba values (5)
    at Decorator.concreteAssert (/Users/beneb/projects/colormin/node_modules/empower-core/lib/decorator.js:95:20)
    at Object.decoratedAssert [as same] (/Users/beneb/projects/colormin/node_modules/empower-core/lib/decorate.js:48:30)
    at Test.fn (/Users/beneb/projects/colormin/dist/__tests__/index.js:43:7)
    at tryCatcher (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/util.js:11:23)
    at Object.gotValue (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/reduce.js:145:18)
    at Object.gotAccum (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/reduce.js:134:25)
    at Object.tryCatcher (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/util.js:11:23)
    at Promise._settlePromiseFromHandler (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/promise.js:489:31)
    at Promise._settlePromise (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/promise.js:546:18)
    at Promise._settlePromiseCtx (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/promise.js:583:10)
    at Async._drainQueue (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/async.js:130:12)
    at Async._drainQueues (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/async.js:135:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/beneb/projects/colormin/node_modules/bluebird/js/release/async.js:16:14)
```

Test is simply:

```js
import test from 'ava';
import min from '..';

test('should return the smallest colour', t => {
  t.same(min('rgba(100%,64.7%,0%,.5)'), 'invalid', 'should convert percentage based rgba values (5)');
});
```

--- a/lib/reporters/mini.js
+++ b/lib/reporters/mini.js
@@ -82,7 +82,7 @@ MiniReporter.prototype.finish = function () {
 	var i = 0;
 
 	if (this.failCount > 0) {
-		this.api.tests.forEach(function (test) {
+		this.api.errors.forEach(function (test) {
 			if (!test.error || !test.error.message) {
 				return;
 			}
@@ -90,10 +90,17 @@ MiniReporter.prototype.finish = function () {
 			i++;
 
 			var title = test.error ? test.title : 'Unhandled Error';
-			var description = test.error ? beautifyStack(test.error.stack) : JSON.stringify(test);
+			var description;
+
+			if (test.error) {
+				description = '  ' + test.error.message;
+				description += '\n  ' + beautifyStack(test.error.stack);
+			} else {
+				description = JSON.stringify(test);
+			}
 
 			status += '\n\n  ' + chalk.red(i + '.', title) + '\n';
-			status += '  ' + chalk.red(description);
+			status += chalk.red(description);
 		});
 	}
 

--- a/test/reporters/mini.js
+++ b/test/reporters/mini.js
@@ -137,7 +137,7 @@ test('results with errors', function (t) {
 	reporter.failCount = 1;
 
 	reporter.api = {
-		tests: [{
+		errors: [{
 			title: 'failed',
 			error: new Error('failure')
 		}]
@@ -149,7 +149,8 @@ test('results with errors', function (t) {
 	t.is(output[1], '  ' + chalk.red('1 failed'));
 	t.is(output[2], '');
 	t.is(output[3], '  ' + chalk.red('1. failed'));
-	t.match(output[4], /Error: failure/);
-	t.match(output[5], /Test\.test/);
+	t.match(output[4], /failure/);
+	t.match(output[5], /Error: failure/);
+	t.match(output[6], /Test\.test/);
 	t.end();
 });

